src/components/Header/index.tsx

```diff
@@ -261,7 +261,7 @@ function Header() {
         newSwprBalance={newSwprBalance}
         stakedAmount={stakedTokenAmount?.toFixed(3)}
         singleSidedCampaignLink={
-          data && !loading ? `/rewards/${data.stakeToken.address}/${data.address}/singleSidedStaking` : undefined
+          data && !loading ? `/rewards/single-campaign/${data.stakeToken.address}/${data.address}` : undefined
```

Comment:
This makes more sense. Should we consider adding a redirect from old pages to new ones?

Reply:
old routes are little buggy since it matches campaign also since the last part is dynamic. Are you concerned about someone has bookmarked it?

Reply:
But old routes will redirect them back to rewards since it doesn't match any routes